workos/event_objects/directory_user.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional, List
+from enum import Enum
+from workos.utils.types import JsonDict
+
+
+class DirectoryUserRole:
+    def __init__(self, attributes: JsonDict) -> None:
+        self.slug: str = attributes["slug"]
+
+
+class DirectoryUserEmail:
+    def __init__(self, attributes: JsonDict) -> None:
+        self.type: Optional[str] = attributes.get("type")
+        self.value: Optional[str] = attributes.get("value")
+        self.primary: Optional[bool] = attributes.get("primary")
+
+
+class DirectoryUserState(Enum):
+    ACTIVE = "active"
+    INACTIVE = "inactive"
+
+
+class DirectoryUserEvent:
+    def __init__(self, attributes) -> None:
+        self.id: str = attributes["id"]
+        self.name: str = attributes["name"]
```

Comment:
What's name here? I don't see it in our docs.

Reply:
Ooh might be a copy/paste error when I was cleaning up. Good catch